examples/01-advanced-postprocessing/Maxwell2D_Transient.py

```diff
@@ -1,12 +1,9 @@
 """
 
 Maxwell 2D Analysis
---------------------------------------------
-This tutorial shows how you can use PyAedt to create a project in Maxwell2D
-and run a transient simulation.
-This example needs PyVista, numpy and matplotlib, to be installed on the
-machine to provide advanced post processing features.
-This example runs on Windows only using CPython.
+-------------------
+This example shows how you can use PyAedt to create a project in
+in Maxwell 2D and run a transient simulation.
 
 To provide the advanced postprocessing features needed for this example, Matplotlib, NumPy, and 
```

Comment:
You could consider providing links to these libraries (pypi or docs)

Reply:
@akaszynski Let me know if my latest modifications look good to you.

Reply:
Please see my comments.